chapters/helloworld/index.rst

```diff
@@ -2,8 +2,18 @@
 Jina "Hello, World!" 👋🌍
 *************************
 
+.. meta::
+   :description: Jina "Hello, World!"s
+   :keywords: Jina, hello world
 
-As a starter, we invite you to try Jina's **Hello, World** - a simple demo of image neural search for `Fashion-MNIST <https://hanxiao.io/2018/09/28/Fashion-MNIST-Year-In-Review/>`_. You will see how to index and query the downloaded dataset. And you will see the retrieved results in a new webpage.
+.. note:: This guide expects you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://docs.jina.ai/chapters/101/index.html>`_ first.
```

Comment:
We're saying this in multiple documents (e.g. 102, some tutorials). Perhaps we should have some standard boilerplate snippet folks can copy/paste (or even embed if the docs platform supports it)

Means less rooms for typos, ensures link consistent, eases maintenance

BTW, we often use http://101.jina.ai at the moment

Reply:
@bwanglzu does the platform support embeds?